man/osg-incommon-cert-request.1

```diff
@@ -0,0 +1,75 @@
+.TH osg-incommon-cert-request 1
+.SH NAME
+osg-incommon-cert-request \- retrieves host and service certificates from the InCommon CA
+.SH SYNOPSIS
+.B osg-incommon-cert-request
+.RI [ OPTION ]...
+.SH DESCRIPTION
+.B osg-incommon-cert-request
+retrieves host and service certificates from the InCommon CA. 
+It requires an Incommon user account authorized as a Department 
+Registration Authority to issue InCommon IGTF Server CA certificates  
+with SSL auto approval. 
+It authenticates the requestor with an InCommon client certificate.
+.PP
+It allows bulk retrieval of certificates and keys. 
+.SH OPTIONS
+.PP
+.TP
+.BR \-\-version
+Show the program's version number and exit.
+.TP
+.BR \-h, \ \-\-help
+Show a help message and exit.
+.TP
+.BR \-\-debug
+Write debug output to stdout.
+.BR \-t, \-\-test
+Testing mode: test connection to InCommon API but does not request certificates.
+.TP
+.BR \-u, \-\-username
+Specify requestor's InCommon username/login.
```

Comment:
It would be good to mention that this is typically an email address, I think.

Reply:
Done